docs/pages/setup/reference/cli.mdx

```diff
@@ -1431,7 +1431,7 @@ $ tctl top http://127.0.0.1:3000 5s
 
 ### tctl version
 
-Print cluster version:
+Print client version:
```

Comment:
```suggestion
Print the version of your `tctl` binary:
```
To make this slightly more precise

Reply:
Sorry for the holdup! Updated this and tsh's equivalents for consistency.